modules/core/src/lib/deck-picker.js

```diff
@@ -121,15 +141,35 @@ export default class DeckPicker {
     // Create a frame buffer if not already available
     if (!this.pickingFBO) {
       this.pickingFBO = new Framebuffer(gl);
+      if (Framebuffer.isSupported(gl, {colorBufferFloat: true})) {
```

Comment:
We should use `hasFeatures(gl, FEATURES.COLOR_ATTACHMENT_RGBA32F)` to cover all cases, on iOS, float texture is supported but rendering to it fails (unlike Mac) , I have more details in https://github.com/uber/luma.gl/issues/1256

Reply:
I'll wait for this to be resolved in luma since we are on the alpha track.